cmake/Modules/FindwxWidgetsTB.cmake

```diff
@@ -497,13 +497,13 @@ if(wxWidgets_FIND_STYLE STREQUAL "win32")
     if(MINGW)
       set(WX_LIB_DIR_PREFIX gcc)
     else()
-	  if(CMAKE_GENERATOR STREQUAL "Visual Studio 10 2010")
+	  if(CMAKE_VS_PLATFORM_TOOLSET STREQUAL "v100")

```

Comment:
Oh, I overlooked that. But I think with a comment, this would be clear enough, and more future proof?

    STRING(REGEX REPLACE "v(\d+)(?:_xp)?" "vc\\1" WX_LIB_DIR_PREFIX ${CMAKE_VS_PLATFORM_TOOLSET})